requirements.in

```diff
@@ -32,3 +32,4 @@ yarl==1.9.2
 zope.event==5.0
 zope.interface==6.0
 requests==2.31.0
+eth-utils==2.2.2
```

Comment:
Can you add that to pyproject.toml as well else this dependency will not exists in the pip package